### DISCUSSION TITLE
FYI: GitHub link in docs is going to wrong repository

### DISCUSSION THREAD

**Discussion by floer32** [crxjs/chrome-extension-tools#241] [General]
Hey, just wanted to let you know that on https://www.extend-chrome.dev/ the little GitHub icon in the right corner is linking to a different repository, maybe just leftover from the SSG or a template. I think it's intended to link to https://github.com/extend-chrome/rollup-plugin-chrome-extension

**Comment by floer32** [comment]
@jacksteamdev

**Reply by jacksteamdev** [reply]
@hangtwenty Thanks for the heads up!
